qutebrowser/browser/network/qutescheme.py

```diff
@@ -159,22 +159,37 @@ def qute_version(_win_id, _request):
 
 @add_handler('plainlog')
 def qute_plainlog(_win_id, _request):
-    """Handler for qute:plainlog. Return HTML content as bytes."""
+    """Handler for qute:plainlog. Return HTML content as bytes.
+
+    An optional query parameter specifies the min log level to print.
+    For example, qute://log?level=warning prints warnings and errors.
+    Level can be one of: vdebug, debug, info, warning, error, critical.
+    """
     if log.ram_handler is None:
         text = "Log output was disabled."
     else:
-        text = log.ram_handler.dump_log()
+        query = QUrlQuery(_request.url().query())
+        level = query.queryItemValue('level')
```

Comment:
I'd prefer [`urllib.parse.parse_qs()`](https://docs.python.org/3.5/library/urllib.parse.html#urllib.parse.parse_qs) here (see [Python and Qt objects](https://github.com/The-Compiler/qutebrowser/blob/master/CONTRIBUTING.asciidoc#python-and-qt-objects) in CONTRIBUTING)